### DISCUSSION TITLE
Unlock Authenticator with U2F

### DISCUSSION THREAD

**Discussion by RomLecat** [Authenticator-Extension/Authenticator#628] [Ideas]
**Describe the feature you want:**
Use an U2F hardware key to unlock access to Authenticator.

**Why do you want this feature in Authenticator?**
It would increase security for websites/services that does not support U2F while supporting TOTP keys.
This way we could conveniently unlock access to the OTP codes using an U2F key.

**Comment by Sneezry** [comment]
We had investigated the solution to unblock Authenticator with U2F. Because Authenticator is a browser extension built with web technology, we can only use web tech to intigate with U2F. Unfortunately, web tech only supports authorization with U2F but not data encription. Because we do not host a server to store user data, authorization doesn't apply to this scenario.

See https://github.com/Authenticator-Extension/Authenticator/pull/496
